tests/model/test_bugscache.py

```diff
@@ -127,38 +124,6 @@ def test_bug_properties(transactional_db, sample_bugs):
     assert set(suggestions['open_recent'][0].keys()) == expected_keys
 
 
-# @pytest.mark.parametrize(("search_term", "exp_bugs"), BUG_SEARCHES)
-# def test_for_empty_search_query(transactional_db, sample_bugs, search_term, exp_bugs):
-#     """
-#     Check Bugscache search strings aren't empty when we query with them
-
-#     This catches a bug introduced as part of the fix for Bug 1469777.
-#     """
-#     bug_list = sample_bugs['bugs']
-#     fifty_days_ago = datetime.now() - timedelta(days=50)
-#     # Update the last_change date so that all bugs will be placed in
-#     # the open_recent bucket, and none in all_others.
-#     for bug in bug_list:
-#         bug['last_change_time'] = fifty_days_ago
-#     _update_bugscache(bug_list)
-
-#     def get_search_query(sql):
-#         search_query, _, _ = sql.partition('\\"\' IN BOOLEAN MODE')
-#         _, _, search_query = search_query.partition('AGAINST (\'\\"')
```

Comment:
What I needed to do to get this test to pass is remove the `'\\"\'` syntax in the two above lines, because I had removed `search_term_fulltext = '"%s"' % sanitised_term` in bugscache.search in #7100 to avoid hitting the mysql bug from that not-good-very-bad-database-crashing string. 

BUT, these tests aren't really needed because it appears the reason for them was to safeguard against someone making a mistake by flipping the string and replacement in `re.sub` and then querying with an empty string, as someone [had done previously](https://bugzilla.mozilla.org/show_bug.cgi?id=1469777#c22). However, since regex that function has been moved into `sanitized_search_term`, which we now have a unit test for, any mistake like that is likely to be caught.